components/channel_view/channel_view.tsx

```diff
@@ -118,6 +124,18 @@ export default class ChannelView extends React.PureComponent<Props, State> {
         if ((this.props.showNextSteps || this.props.showNextStepsTips) && !this.props.isOnboardingHidden) {
             this.props.actions.setShowNextStepsView(true);
         }
+
+        const {currentUserId, onIntroTutorialScreen, actions} = this.props;
+        if (onIntroTutorialScreen) {
+            const preferences = [{
+                user_id: currentUserId,
+                category: Preferences.TUTORIAL_STEP,
+                name: currentUserId,
+                value: '1',
```

Comment:
what does this '1' value represent? would it make sense to use it as a constant value?

Reply:
I didn't want to mess with the type definition since that would involve touching so many other parts of the code that use it.